### PR TITLE
[AzureMonitorDistro] prep release 1.3.0 beta2

### PR DESCRIPTION
**Pull request by TimothyMothra**
Note: Exporter beta 2 is releasing right now. The builds here will fail until this publish is complete.

### CODE REVIEW

File: sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/Azure.Monitor.OpenTelemetry.AspNetCore.csproj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (rajkumar-rangaraj) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/Azure.Monitor.OpenTelemetry.AspNetCore.csproj:1]: What has changed here?

Comment (TimothyMothra) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/Azure.Monitor.OpenTelemetry.AspNetCore.csproj:1]: This usually means the file encoding changed.
When Powershell scripts edit our csproj files, it seems to apply one encoding. And then when we edit files in Visual Studio it sometimes applies a different encoding.

**Review by rajkumar-rangaraj — APPROVED** [review]

**Review by jsquire — APPROVED** [review]

**Pull request merged**